### PR TITLE
30 fully document getty ingest process

### PR DESCRIPTION
**Pull request by cwulfman**

### CODE REVIEW

File: doc/how_to_build_site.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  1. a TEI document for each ciconum;
+  2. and a TEI document for each dclnum.
+  
+
+To generate these files, run the `convert_getty_to_tei` and the `items_generator` Rake tasks:
+
+   ```shell
+   rake convert_getty_to_tei
+   rake items_generator
```
Comment (escowles) [inline doc/how_to_build_site.md:57]: Looks like the rake file is named `items_generator`, but the rake task is named `generate_item_files` — do you want to rename the task?

**Comment by escowles** [comment]
I was able to walk through the commands, but at the end the static site had all of the digital objects removed, so I think there is something not quite right somewhere in the process.

**Comment by cwulfman** [comment]
> I was able to walk through the commands, but at the end the static site had all of the digital objects removed, so I think there is something not quite right somewhere in the process.

There's a bug in the code that generates the dclnum index; I'll fix it.

**Comment by cwulfman** [comment]
I've drafted a diagram (https://drive.google.com/file/d/1xvnS9i7-aJJMEtBUrVEykstLg0qgmBjX/view?usp=drive_link) to show this workflow; please feel free to make it better.

From the flowchart, it appears the cicoindex.json file may be vestigial. At some point I'll verify that and, if so, remove the file and the code.

**Review by escowles — APPROVED** [review]
👍🏻 great — now the process works for me and generates only very minor changes (looks like a couple of digital objects were removed) that we can fold into the planned update in the fall.

**Pull request merged**